backend/ee/danswer/server/query_and_chat/chat_backend.py

```diff
@@ -51,6 +56,24 @@ def remove_answer_citations(answer: str) -> str:
     return re.sub(pattern, "", answer)
 
 
+@router.post("/test-standard-answer")
```

Comment:
It's a valid use case, doesn't have to be for test right? I think we can just call it get-standard-answer or something

Technically to be correct about it, I think this is better as a GET since it doesn't change the state of anything, but I don't particularly care. I know we have it wrong in other places as well.